integration/tests/cook/test_basic.py

```diff
@@ -2655,3 +2655,26 @@ def get_debug_status_code():
             self.assertEqual(404, missing_resp.status_code, missing_resp.text)
         finally:
             util.kill_jobs(self.cook_url, [job_uuid], assert_response=False)
+
+
+    @unittest.skipUnless(util.supports_mesos_containerizer_images(), "Requires support for docker images in mesos containerizer")
+    def test_mesos_containerizer_image_support(self):
+        job_uuid, resp = util.submit_job(self.cook_url, container={'type': 'mesos', 'mesos': {'image': 'alpine'}})
+        try:
+            self.assertEqual(201, resp.status_code, resp.text)
+            job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
```

Comment:
Do we actually need to wait until the job completes in order to harvest the container info from mesos?

Reply:
Maybe we can just use `wait_for_instance`, which returns the instance, and then we can avoid the `job['instances'][0]` in this test?

Reply:
I had intended to ensure that the job completed successfully (not failing due to container error or something similar) but hadn't actually done that. Added the assertion.